src/main/java/org/epamTask/BuildingType.java

```diff
@@ -0,0 +1,10 @@
+package org.epamTask;
+
+public enum BuildingType {
+    UNIVERSITY,
```

Comment:
please add private field to enum
private String name
and change this name like UNIVERSITY("University")